wiki/osu!_File_Formats/Db_(file_format)/fr.md

```diff
@@ -50,73 +50,74 @@ Quelques types de données sont spécifiques à osu!.db, les voici:
 | Booléen (Boolean) | Est définit par false (faux) si votre compte à été banni ou à été restreint. |
 | Date (DateTime) | La date à laquelle le compte osu! ne sera plus restreint |
 | Chaîne de caractère (string) | Nom du joueur |
-| Entier (Int) | 	Nombre de beatmaps installés |
+| Entier (Int) | Nombre de beatmaps installées |
 | Beatmaps* (nom universel) | Données des beatmaps |
 | Entier (Int) | Utilisation inconnue, visiblement définie par 4 tout le temps |
 
 ### Informations sur les beatmaps
 
 | Type de donnée | Description |
 | --------- | ----------- |
-| Chaîne de caractère (string) | Nom de l'artiste |
-| Chaîne de caractère (string) | Nom de l'artiste en Unicode |
-| Chaîne de caractère (string) | Titre de la musique |
-| Chaîne de caractère (string) | Titre de la musique en Unicode |
-| Chaîne de caractère (string) | Nom du mappeur |
-| Chaîne de caractère (string) | Nom de la difficulté (exemple: Hard, Insane, etc.) |
-| Chaîne de caractère (string) | Nom du fichier audio |
-| Chaîne de caractère (string) | Cryptage en MD5 de la beatmap |
-| Chaîne de caractère (string) | Nom (nom anglais) of the .osu file corresponding to this beatmap |
```

Comment:
"Nom (nom anglais) of the .osu file corresponding to this beatmap"

non traduit ? "Nom du fichier .osu représentant la beatmap"